src/components/index.js

```diff
@@ -1,7 +1,8 @@
 import { TextOverMediaCard } from './card/TextOverMediaCard';
 import { BasicCard } from './card/BasicCard';
-import {HorizontalCard} from './card/HorizontalCard'
+import { HorizontalCard } from './card/HorizontalCard'
 import { RatingPalleteIcon } from './rating/RatingPalleteIcon';
 import { ButtonPrimary, ButtonSecondary, OutlineButtonPrimary, OutlineButtonSecondary } from './buttons'
+import {Sidebar} from './sidebar/Sidebar';
```

Comment:
do export on the same line 
`export {HorizontalCard} from './card/HorizontalCard'`